common/components/message/_message.scss

```diff
@@ -87,3 +93,22 @@
   border-width: 0 0 0 5px;
   background-color: govuk-colour("light-grey");
 }
+
+.app-message--temporary {
+  background-color: govuk-colour("blue");
+  border-width: 0;
+  color: govuk-colour("white");
+  padding: $_message-padding + 5;
```

Comment:
should we be aligning white space and not using magic numbers?

Reply:
Do you mean the variable or the `5`? The 5 is the border width. 

Maybe that could be a variable so it's easier to understand.

Reply:
> Maybe that could be a variable so it's easier to understand.

This sounds good